stsci.skypac/meta.yaml

```diff
@@ -10,14 +10,15 @@ about:
 build:
     number: {{ number }}
     preserve_egg_dir: 'yes'
-
+    skip: true  # [py2k]
```

Comment:
Only @jhunkeler would know if the syntax is correct. Can there be a space after `#`? The original example he gave me did not have a space but maybe that does not matter.

Reply:
Syntax should be fine (I think).

Reply:
@pllim 

> The original example he gave me did not have a space...

Why did you add it then???

Reply:
I did? Where?

Reply:
<img width="449" alt="retire_py2k" src="https://user-images.githubusercontent.com/5985250/51406373-fc78e780-1b26-11e9-9d5c-f0e011418af6.png">

Reply:
🤣

Reply:
Haha... OK... Then LGTM!